journal-week-4-tutorial.md

```diff
@@ -0,0 +1,7 @@
+**Self Closing Tags**: Tags that close themselves. There's nothing that would go inbetween the tags so there's no need for a closing tag (</....>). All the information is within the the tag itself.
+
+Some examples of self closing tags are:
+
+* **[link] (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link)**: a tag the links the html to another document, usually a CSS folder
+* **[meta] (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta)**: represents any metadata information that cannot be represented by one of the other HTML meta-related elements (<base>, <link>, <script>, <style> or <title>).
+
```

Comment:
You need to wrap those tags in backticks so they don't get treated like HTML. More words better.